mkt/extensions/views.py

```diff
@@ -85,6 +86,15 @@ class ExtensionViewSet(CORSMixin, MarketplaceView,
     queryset = Extension.objects.all()
     serializer_class = ExtensionSerializer
 
+    def filter_queryset(self, qs):
+        if self.action == 'list':
+            # The listing API only allows you to see extensions you've
+            # developed.
+            if not self.request.user.is_authenticated():
```

Comment:
Is there any reason for this? I presume we don't allow anonymous submission, and we don't allow anonymous listing, so we could just remove `RestAnonymousAuthentication`.